### ISSUE TITLE
Definition of 'occurs at' to be broadened

### ISSUE THREAD

**Issue by neilotte** [CommonCoreOntology/CommonCoreOntologies#144]
I believe 'occurs at' should read (additions in BF):

"x occurs_at y iff x is an instance of Process and y is an instance of **_either_** Site **_or Spatial Region_**, such that x occurs in y."

**Comment by swartik** [comment]
If this is correct, then the definition of the inverse property (is site of) should read:

"x is_site_of y iff x is an instance of _either_ Site _or Spatial Region_ and y is an instance of Process, such that y occurs in x."

**Comment by jonathanvajda** [comment]
> If this is correct, then the definition of the inverse property (is site of) should read:
> 
> "x is_site_of y iff x is an instance of _either_ Site _or Spatial Region_ and y is an instance of Process, such that y occurs in x."

This follows, since this inverse relation has also been updated to be more generic/inclusive too. However, wouldn't this mean that this relation is not ideally named?

**Comment by neilotte** [comment]
@jonathanvajda  Indeed. I'll take a note to update the label following the switch to opaque identifiers.

**Comment by neilotte** [comment]
@johnbeve  Can you take a look at the comments on this PR when you have a moment? 

https://github.com/CommonCoreOntology/CommonCoreOntologies/pull/321

**Comment by neilotte** [comment]
@johnbeve  Scratch that. Let's revisit under 2.0.

**Comment by neilotte** [comment]
Moving to 2.1.

**Comment by alanruttenberg** [comment]
occurs in is a BFO relationship. Why is it being modified here?
